screens/Scan/SharingStatusModal.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {useTranslation} from 'react-i18next';
+import {Theme} from '../../components/ui/styleUtils';
+import {Modal} from '../../components/ui/Modal';
+import {Pressable, Dimensions} from 'react-native';
+import {Button, Column, Row, Text} from '../../components/ui';
+import testIDProps from '../../shared/commonUtil';
+import {Icon} from 'react-native-elements';
+
+export const SharingStatusModal: React.FC<SharingStatusModalProps> = props => {
+  const {t} = useTranslation('ScanScreen');
+
+  return (
+    <React.Fragment>
+      <Modal
+        isVisible={props.isVisible}
+        showClose={false}
+        {...testIDProps(props.testId)}>
+        <Column
+          margin="64 0"
+          crossAlign="center"
+          style={Theme.SelectVcOverlayStyles.sharedSuccessfully}>
+          {props.image}
+          <Text
+            testID="sharingStatusTitle"
+            margin="20 0"
+            style={Theme.TextStyles.bold}
+            size={'large'}>
+            {props.title}
+          </Text>
+          <Text
+            testID="sharingStatusMessage"
+            align="center"
+            margin="0 33 0 33"
+            style={Theme.TextStyles.regular}
+            color={Theme.Colors.statusMessage}>
+            {props.message}
+          </Text>
+        </Column>
+        {props.status === 'successfullyShared' ? (
+          <Row
+            align="space-evenly"
+            style={{marginBottom: Dimensions.get('screen').height * 0.06}}>
+            <Pressable testID="successfulVcSharedHomeIcon">
+              <Icon
+                name="home"
+                color={Theme.Colors.Icon}
+                size={33}
+                containerStyle={
+                  Theme.SelectVcOverlayStyles.sharedSuccessfullyIconStyle
+                }
+                onPress={props.goToHome}
+              />
+              <Text align="center" weight="bold">
+                {t('status.accepted.home')}
+              </Text>
+            </Pressable>
+            <Pressable testID="successfulVcSharedHistoryIcon">
+              <Icon
+                name="history"
```

Comment:
Suggestion: We already have history svg image in svg.tsx, then we can reuse that in here as well instead of icon

Reply:
Followed same as above.

Reply:
I have followed that. However, I couldn't able to add styles for SVG's separately. Attached screenshot.
[success screen](https://github.com/mosip/inji/assets/106086523/8d22aa6e-384f-4ae6-81dd-83f489eeea99)